### PR TITLE
Release Notes for May 2024

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:
* **Release Notes for the WebView2 SDK** > **1.0.2535.41**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/index?branch=pr-en-us-3136#10253541
* **Release Notes for the WebView2 SDK** > **1.0.2584-prerelease**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/index?branch=pr-en-us-3136#102584-prerelease
* **Archived Release Notes for the WebView2 SDK**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes/archive?branch=pr-en-us-3136 - moved Dec, Nov, & Oct RelNotes from the above webpage to here

AB#50304234

### CODE REVIEW

File: microsoft-edge/webview2/release-notes/index.md [hunk 1]
```diff
@@ -56,16 +57,142 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
...
+
+<!-- ------------------------------ -->
+#### Bug fixes
+
+
+<!-- ---------- -->
+###### Runtime-only
+
+* Fixed a bug where if the `LaunchingExternalURIScheme` event handler is attached, and the **always remember** checkbox is enabled, and the user selects this checkbox, the dialog is incorrectly displayed again. 
+* Fixed an issue where text edit controls in visual hosting would duplicate IME input when losing and regaining focus.<!-- todo: when losing OR regaining focus? -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes/index.md:132]: AND, or OR?
* Fixed an issue where text edit controls in visual hosting would duplicate IME input when losing and regaining focus.
* Fixed an issue where text edit controls in visual hosting would duplicate IME input when losing or regaining focus.
@victorhuangwq @peiche-jessica

Comment (victorhuangwq) [reply microsoft-edge/webview2/release-notes/index.md:132]: Losing AND regaining I think. @peiche-jessica  this is the corresponding PR: https://microsoft.visualstudio.com/Edge/_git/a45cf45f-f6db-438b-baf7-da6b3de589ba/commit/c20fd2234da151520aaa281962d9ad8a67d548b8?tab=details

Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes/index.md]: changed to:
```
* Fixed an issue where text edit controls in visual hosting would duplicate IME input 
when losing and then regaining focus.
```
@peiche-jessica @victorhuangwq

**Review by victorhuangwq — APPROVED** [review]
LGTM. Just need to edit the dates to match the actual release date. Which would likely be 28th of May, since 27th is Memorial day

**Comment by mikehoffms** [comment]
> LGTM. Just need to edit the dates to match the actual release date. Which would likely be 28th of May, since 27th is Memorial day

updated to 28

**Review by peiche-jessica — APPROVED** [review]

**Pull request merged**